src/app.php

```diff
@@ -27,6 +27,7 @@
     // add custom globals, filters, tags, ...
 
     return $twig;
-});
+}
```

Comment:
albo robisz wciecie dla } albo robisz }); bo to co napisales nie jest zgodne z PSR

Reply:
no właśnie tu mnie zmyliły uwagi code sniffer...Będę się trzymał });

Reply:
jezeli code sniffer cos ci pisal to napewno bylo zle ale cos innego :)